pyzeebe/grpc_internals/zeebe_job_adapter.py

```diff
@@ -36,6 +36,7 @@ async def activate_jobs(
         request_timeout: int,
     ) -> AsyncGenerator[Job, None]:
         try:
+            grpc_request_timeout = request_timeout / 1000 * 2 if request_timeout > 0 else 20
```

Comment:
Would move 20 to a const named `DEFAULT_GRPC_REQUEST_TIMEOUT`.

Reply:
done